ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1099,6 +1099,86 @@ def test_ipahealthcheck_ds_riplugincheck(self, update_riplugin):
             assert check["result"] == "WARNING"
             assert warn_msg in check["kw"]["msg"]
 
+    @pytest.fixture
+    def update_logging(self):
+        """
+        Fixture disables nsslapd-logging-hr-timestamps-enabled
+        parameter and reverts it back
+        """
+        ldap = self.master.ldap_connect()
+        dn = DN(
+            ("cn", "config"),
+        )
+        entry = ldap.get_entry(dn)  # pylint: disable=no-member
+        entry.single_value["nsslapd-logging-hr-timestamps-enabled"] = 'off'
+        ldap.update_entry(entry)  # pylint: disable=no-member
+
+        yield
+
+        entry = ldap.get_entry(dn)  # pylint: disable=no-member
+        entry.single_value["nsslapd-logging-hr-timestamps-enabled"] = 'on'
+        ldap.update_entry(entry)  # pylint: disable=no-member
+
+    def test_ipahealthcheck_ds_configcheck(self, update_logging):
+        """
+        This testcase ensures that ConfigCheck displays warning
+        when high resolution timestamp is disabled.
+        """
+        warn_msg = (
+            "nsslapd-logging-hr-timestamps-enabled changes the "
+            "log format in directory server "
+        )
+        returncode, data = run_healthcheck(
+            self.master,
+            "ipahealthcheck.ds.config",
+            "ConfigCheck",
+        )
+        assert returncode == 1
+        for check in data:
+            assert check["result"] == "WARNING"
+            assert check["kw"]["key"] == "DSCLE0001"
+            assert 'cn=config' in check["kw"]["items"]
+            assert warn_msg in check["kw"]["msg"]
```

Comment:
The lib389 config check also looks at the passwordStorageScheme and nsslapd-rootpwstoragescheme to require them to be in ['SSHA512', 'PBKDF2_SHA256']

Reply:
@rcritten a new test has been added to address the above scenario: test_ds_configcheck_passwordstorage